src/katgpucbf/fgpu/main.py

```diff
@@ -105,7 +103,7 @@ class _NarrowbandOutputDict(_OutputDict, total=False):
     pass_bandwidth: float
 
 
-def _parse_stream(value: str, kws: _OD, field_callback: Callable[[_OD, str, str], None]) -> None:
+def _parse_stream[OD: _OutputDict](value: str, kws: OD, field_callback: Callable[[OD, str, str], None]) -> None:
```

Comment:
I ~understand why you've done away with the underscore notation in previous cases. Here, however, still refers to a class whose name contains it - `_OutputDict`. Either way, do you reckon that distinction isn't as important anymore?

Reply:
The difference is that `_OutputDict` exists at module scope, so it could be referenced from other modules and the `_` prefix is a warning to those modules not to do that. The `OD` here doesn't exist outside of this function, so it is already private and doesn't need a prefix to tell other users "hands off" because they can't refer to it anyway.